src/setup-binary.ts

```diff
@@ -32,46 +32,46 @@ export async function fetchBinary(versionSpec: string): Promise<string> {
 
   let binaryPath: string;
 
-  core.info(
-    `Finding an application version that matches version spec '${versionSpec}'.`
-  );
+  core.info(`Finding release that matches ${versionSpec}.`);
   let release = await hc.getRelease(BINARY_NAME, versionSpec, USER_AGENT);
 
-  const version = release.version;
-  core.info(`Found ` + BINARY_NAME + ` ${version}.`);
+  const {version} = release;
+  let nameAndVersion = BINARY_NAME + ` ` + version;
```

Comment:
should these be let?

Reply:
Unlikely; they should be `const` - updated